lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1759,6 +1759,18 @@ definitions:
         type: boolean
         example: true
         default: false
+      enable_5G_feature_set:
+        description: Enables 5G Standalone (SA) for pipelined at a network level
+        properties:
+          enable:
+            type: boolean
+            example: true
+            default: false
+      converged_access:
+        description: Enables 5G Standalone (SA) for sessiond at a network level
+        type: boolean
+        example: true
+        default: false
```

Comment:
you do not need to add two more fields on swagger API to enable 5G feature. `enable_converged_core` is sufficient to figure out that you need to enable 5G services across all AGW services.

Reply:
Sure, @ulaskozat . Will do that change.

Reply:
I also suggest to change the name `enable_converged_core` to `enable_5G_features`.

Reply:
Sure Ulas, Will do that change.